Automattic/course
# Style the comment thread and reply form to look closer to design

Implements https://github.com/Automattic/sensei-theme/issues/6

### Changes proposed
Changes the following to match the design

- Comment reply form input sizes, borders and orders of all input fields

- Form input fields structure in desktop and mobile
- All label and description font sizes
- Background color for input elements
- Button font sizes
- URL designs
- Author title font and spacing
- Author avatar spacing and border setting
- Comment reply date spacing and position
- Comment reply body spacing and position
- Edit link
- Reply link
- Reply text and Reply cancel button
- Spacing between the reply threads and inner elements
- Alignment of avatar, title and content elements

### Testing instruction

- Add a post
- Allow commenting to that post
- Go to the frontend and add a couple of comments and replies
- Make sure they look similar to the design

### Screenshot
Desktop
<img width="835" alt="image" src="https://user-images.githubusercontent.com/6820724/195433948-8319b62f-48aa-4a5a-8bdd-e4ed2455bc9f.png">

Mobile
<img width="410" alt="image" src="https://user-images.githubusercontent.com/6820724/195434086-b583f636-18c2-41b6-b2a5-f39d8254eb6a.png">
